test/unit/ui/marker.test.js

```diff
@@ -16,31 +16,60 @@ function createMap() {
 }
 
 test('Marker', (t) => {
+
     t.test('constructor', (t) => {
-        const el = window.document.createElement('div');
-        const marker = new Marker(el);
-        t.ok(marker.getElement(), 'marker element is created');
+        const marker = new Marker();
+        t.ok(marker, 'marker is created');
         t.end();
     });
 
-    t.test('default marker', (t) => {
+    t.test('default marker without element', (t) => {
         const marker = new Marker();
-        t.ok(marker.getElement(), 'default marker is created');
-        t.ok(marker.getOffset().equals(new Point(0, -14)), 'default marker with no offset uses default marker offset');
+        t.ok(marker.getElement(), 'marker returns default element');
+        t.end();
+    });
+
+    t.test('default marker with element', (t) => {
```

Comment:
It's not as clear as it could be, but I used the term "default marker" to mean when you create a marker without passing an element, so it uses the default Marker SVG element instead. So "default marker with element" doesn't make sense in that context.

What do you mean by default marker with element?